typings/localforage.d.ts

```diff
@@ -17,8 +17,12 @@ interface LocalForageOptions extends LocalForageDbInstanceOptions {
 interface LocalForageDbMethodsCore {
     getItem<T>(key: string, callback?: (err: any, value: T | null) => void): Promise<T | null>;
 
+    getMultipleItems<T>(keys: ReadonlyArray<keyof T>, synchronizationKey: string): Promise<Readonly<{...T, synchronizationValue?: string}>>;
```

Comment:
```suggestion
    getMultipleItems<T>(keys: ReadonlyArray<keyof T>, synchronizationKey: string): Promise<Readonly<{value: T, synchronizationValue?: string}>>;
```

same as [here](https://phab.comm.dev/D12836#inline-74725) - we think that it's better and more readable than spreading